templates/web/fixmystreet.com/footer_extra.html

```diff
@@ -37,9 +37,9 @@ <h2 class="mysoc-footer__site-name">FixMyStreet</h2>
                             %]>[% loc("Help") %]</[% c.req.uri.path == '/faq' ? 'span' : 'a' %]></li>
                     </ul>
                     <ul>
-                        <li><a href="/pro/">FixMyStreet Pro</a></li>
+                        <li><a href="https://www.societyworks.org/services/highways/">FixMyStreet Pro</a></li>
                         <li><[% IF c.req.uri.path == '/about/posters' %]span[% ELSE %]a href="/about/posters"[% END
-                            %]>[% loc("FixMyStreet Goodies") %]</[% c.req.uri.path == '/posters' ? 'span' : 'a' %]></li>
+                            %]>[% loc("Community promotional material") %]</[% c.req.uri.path == '/posters' ? 'span' : 'a' %]></li>
```

Comment:
Can we think of anything pithier? Only it makes it take up three lines now, bit odd in comparison to the rest:
![image](https://github.com/mysociety/fixmystreet/assets/154364/f66887c6-c380-47ca-b4d2-453fc413744c)
Or dunno if any other solution.

Reply:
I guess could just say 'Promotional material' which is what the page is called?